src/kv/test/kv_bench.cpp

```diff
@@ -39,7 +39,7 @@ static void serialise(picobench::state& s)
 
   auto& map0 = kv_store.create<std::string, std::string>("map0", SD);
   auto& map1 = kv_store.create<std::string, std::string>("map1", SD);
-  Store::Tx tx;
+  StoreTx tx;
```

Comment:
`StoreTx` looks strange. These are really `ccf::StoreTx`, and should perhaps instead be `ccf::Tx`? But I'm wary of just using `Tx` everywhere (which is possible because all of our tests do `using namespace ccf;`). Maybe its fine. This is a tomorrow problem.

Reply:
I've decided to call this `ccf::Tx`, but including the namespace everywhere this is used for clarity, even where its redundant. So (hopefully), `Tx` is always `kv::Tx<S, D>`, and we can distinguish it from `ccf::Tx` aka `kv::Tx<CCF's serialiser, CCF's deserialiser>`. 2 related points:

1) When we go to everything-in-the-kv-is-a-byte-vector we can probably remove the `<S, D>` in most places?
2) I think we're using the `ccf` namespace in too many places. I've moved the logging app into its own `logging` namespace as a test, we should do this with the other apps. And the unit tests should, where possible, avoid `using namespace ccf` (eg - the kv unit tests should really only test the `kv::` namespace, not the specialisation choices that CCF makes).

Reply:
1. it depends if Map is the wrapper type or not. In practice we will, but users will probably still be looking at UserMap<K, V> where K and V implement serialise()/deserialise() or something along those lines.
2. Agreed, but logging is too similar to one of the internal namespaces.